libdino/src/service/counterpart_interaction_manager.vala

```diff
@@ -66,6 +66,7 @@ public class CounterpartInteractionManager : StreamInteractionModule, Object {
     private void on_chat_marker_received(Account account, Jid jid, string marker, string stanza_id) {
         bool own_marker = account.bare_jid.to_string() == jid.bare_jid.to_string();
         if (own_marker) {
+            if (marker == Xep.ChatMarkers.MARKER_RECEIVED) return;
```

Comment:
Just a *very* minor thing: I think it would be clearer to have this condition switched around - to to the following for MARKER_DISPLAYED and MARKER_ACKNOWLEDGED.

Reply:
Ok, fixed!